inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
is_LIGATURE.h is not needed as it is already covered by inc/utype.h, and is_LIGATURE.h would have to include inc/utype.h within it.

is_LIGATURE.c currently exists because I did not include it in utype.c even though technically speaking, from a makeutype.c point of view, it could have been appended to utype.c

is_LIGATURE.c makes more sense keeping separate from utype.c because (1) it keeps utype.c from growing further (2) adds some sense of clarification between utype.c and is_LIGATURES.c (3) the original code was written by an author that was pro-GPL, and therefore my best guess was to also call this GPL for patch "Moved is_LIGATURES into Unicode library instead of fontforge library" to satisfy the request to add copyright notices (4) even though there is only about 3 lines of original code left from the old is_LIGATURE.c to the new is_LIGATURE.c, there are also additional routines I would like to add under the GPL banner, and keeping it seperate, adds clarity (utype.c==BSD+3, is_LIGATURE.c==GPL).